README.md

```diff
@@ -1,8 +1,30 @@
 # WayOfWorking::Changelog::Keepachangelog
 
-TODO: Delete this and the text below, and describe your gem
+This is a plugin for the [Way of Working](https://github.com/HealthDataInsight/way_of_working) framework. It uses [keep a changelog](https://keepachangelog.com/en/1.1.0/) version 1.1.0. It has a well-defined structure that makes it an accessible format for people and tooling.
 
-Welcome to your new gem! In this directory, you'll find the files you need to be able to package up your Ruby library into a gem. Put your Ruby code in the file `lib/way_of_working/changelog/keepachangelog`. To experiment with that code, run `bin/console` for an interactive prompt.
+To quote from their website:
+
+> Guiding Principles:
+> * Changelogs are for humans, not machines.
+> * There should be an entry for every single version.
+> * The same types of changes should be grouped.
+> * Versions and sections should be linkable.
+> * The latest version comes first.
+> * The release date of each version is displayed.
+> * Mention whether you follow Semantic Versioning.
```

Comment:
Is this something we could enforce, i.e. everything should follow semantic versioning?